Build failure

2025-07-14T04:05:42.2516274Z   CLOUDFLARE_ACCOUNT_ID: ***
2025-07-14T04:05:42.2516624Z ##[endgroup]
2025-07-14T04:05:42.2601051Z $ opennextjs-cloudflare build && opennextjs-cloudflare deploy
2025-07-14T04:05:44.2096432Z 
2025-07-14T04:05:44.2097239Z ┌─────────────────────────────┐
2025-07-14T04:05:44.2097898Z │ OpenNext — Cloudflare build │
2025-07-14T04:05:44.2098504Z └─────────────────────────────┘
2025-07-14T04:05:44.2111353Z 
2025-07-14T04:05:44.2754390Z App directory: /home/runner/work/ryancruz.com/ryancruz.com
2025-07-14T04:05:44.2755085Z Next.js version : 15.3.5
2025-07-14T04:05:44.2759829Z @opennextjs/cloudflare version: 1.5.1
2025-07-14T04:05:44.2761753Z @opennextjs/aws version: 3.7.0
2025-07-14T04:05:44.2762053Z 
2025-07-14T04:05:44.2762540Z ┌─────────────────────────────────┐
2025-07-14T04:05:44.2763178Z │ OpenNext — Building Next.js app │
2025-07-14T04:05:44.2763712Z └─────────────────────────────────┘
2025-07-14T04:05:44.2763965Z 
2025-07-14T04:05:44.2830524Z $ next build
2025-07-14T04:05:44.8956784Z ⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
2025-07-14T04:05:44.9011025Z Attention: Next.js now collects completely anonymous telemetry regarding usage.
2025-07-14T04:05:44.9013326Z This information is used to shape Next.js' roadmap and prioritize features.
2025-07-14T04:05:44.9014387Z You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
2025-07-14T04:05:44.9015142Z https://nextjs.org/telemetry
2025-07-14T04:05:44.9015421Z 
2025-07-14T04:05:45.1053469Z    ▲ Next.js 15.3.5
2025-07-14T04:05:45.1053807Z 
2025-07-14T04:05:46.0305933Z    Creating an optimized production build ...
2025-07-14T04:05:58.6600469Z  ✓ Compiled successfully in 8.0s
2025-07-14T04:05:58.6638641Z    Linting and checking validity of types ...
2025-07-14T04:06:01.4944333Z 
2025-07-14T04:06:01.4944887Z Failed to compile.
2025-07-14T04:06:01.4945128Z 
2025-07-14T04:06:01.4945255Z ./components/Header.tsx
2025-07-14T04:06:01.4945817Z 7:8  Error: 'AirportIcon' is defined but never used.  @typescript-eslint/no-unused-vars
2025-07-14T04:06:01.4946308Z 
2025-07-14T04:06:01.4946461Z ./components/PassportCover.tsx
2025-07-14T04:06:01.4948200Z 25:11  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` or a custom image loader to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
2025-07-14T04:06:01.4951996Z 43:11  Warning: Using `<img>` could result in slower LCP and higher bandwidth. Consider using `<Image />` from `next/image` or a custom image loader to automatically optimize images. This may incur additional usage or cost from your provider. See: https://nextjs.org/docs/messages/no-img-element  @next/next/no-img-element
2025-07-14T04:06:01.4953658Z 
2025-07-14T04:06:01.4954258Z info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/app/api-reference/config/eslint#disabling-rules
2025-07-14T04:06:01.5161654Z error: script "build" exited with code 1
2025-07-14T04:06:01.5208446Z node:internal/errors:984
2025-07-14T04:06:01.5210138Z   const err = new Error(message);
2025-07-14T04:06:01.5211976Z               ^
2025-07-14T04:06:01.5213191Z 
2025-07-14T04:06:01.5214381Z Error: Command failed: bun run build
2025-07-14T04:06:01.5215879Z     at genericNodeError (node:internal/errors:984:15)
2025-07-14T04:06:01.5217416Z     at wrappedFn (node:internal/errors:538:14)
2025-07-14T04:06:01.5218977Z     at checkExecSyncError (node:child_process:891:11)
2025-07-14T04:06:01.5220505Z     at Object.execSync (node:child_process:963:15)
2025-07-14T04:06:01.5222699Z     at buildNextjsApp (file:///home/runner/work/ryancruz.com/ryancruz.com/node_modules/@opennextjs/aws/dist/build/buildNextApp.js:15:8)
2025-07-14T04:06:01.5225608Z     at build (file:///home/runner/work/ryancruz.com/ryancruz.com/node_modules/@opennextjs/cloudflare/dist/cli/build/build.js:45:9)
2025-07-14T04:06:01.5228017Z     at runCommand (file:///home/runner/work/ryancruz.com/ryancruz.com/node_modules/@opennextjs/cloudflare/dist/cli/index.js:39:20)
2025-07-14T04:06:01.5230202Z     at async file:///home/runner/work/ryancruz.com/ryancruz.com/node_modules/@opennextjs/cloudflare/dist/cli/index.js:51:1 {
2025-07-14T04:06:01.5231182Z   status: 1,
2025-07-14T04:06:01.5231457Z   signal: null,
2025-07-14T04:06:01.5231757Z   output: [ null, null, null ],
2025-07-14T04:06:01.5232101Z   pid: 2154,
2025-07-14T04:06:01.5232381Z   stdout: null,
2025-07-14T04:06:01.5232654Z   stderr: null
2025-07-14T04:06:01.5232914Z }
2025-07-14T04:06:01.5233056Z 
2025-07-14T04:06:01.5233192Z Node.js v20.19.3
2025-07-14T04:06:01.5439555Z error: script "deploy" exited with code 1
2025-07-14T04:06:01.5464290Z ##[error]Process completed with exit code 1.

--- a/components/Header.tsx
+++ b/components/Header.tsx
@@ -4,7 +4,6 @@ import { useEffect, useState } from 'react'
 import { motion, AnimatePresence } from 'framer-motion'
 import { useTheme } from './ThemeProvider'
 import ThemeToggle from './ThemeToggle'
-import AirportIcon from './AirportIcon'
 
 export default function Header() {
   const { resolvedTheme } = useTheme()

